apps/nextjs-website/src/pages/404.tsx

```diff
@@ -0,0 +1,29 @@
+import { translations } from '@/_contents/translations';
+import SectionTitle from '@/components/molecules/SectionTitle/SectionTitle';
+import Layout, { LayoutProps } from '@/components/organisms/Layout/Layout';
+import { getProducts } from '@/lib/api';
+import { GetStaticProps, GetStaticPropsResult } from 'next';
+
+export type PageNotFoundProps = LayoutProps;
+
+export const getStaticProps: GetStaticProps<
+  PageNotFoundProps
+> = (): GetStaticPropsResult<LayoutProps> => {
+  return { props: { products: [...getProducts()] } };
+};
+
+export default function PageNotFound({ products }: PageNotFoundProps) {
+  const { pageNotFound } = translations;
+
+  return (
+    <Layout products={products}>
+      <SectionTitle
```

Comment:
Forse userei Abstract in questo caso perché mi sembra un po' troppo schiacciata:
![image](https://github.com/pagopa/developer-portal/assets/39835990/ed25791c-33ef-481c-813d-746e2cdaf704)